packages/internal/src/tagged.d.ts

```diff
@@ -0,0 +1,155 @@
+/** @file adapted from https://raw.githubusercontent.com/sindresorhus/type-fest/main/source/opaque.d.ts */
```

Comment:
What's the license? ah. CC0 Public Domain Dedication. very well.

might be worth noting that.